packages/npm2gpr/index.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require("node-fetch");
+const url1 = "https://registry.npmjs.org/-/v1/search?text=@creately&from=0";
+const url2 = "https://registry.npmjs.org/-/v1/search?text=@creately&from=20";
+
+const getData = async url => {
+  try {
+    const response = await fetch(url);
+    const json = await response.json();
+    //console.log(json.objects.package);
+    arr = [];
+    arr = json.objects;
+    for (var i = 0; i < arr.length; i++) {
+        console.log(arr[i].package.name)
```

Comment:
Do not log the package name. Collect it to an array and return the array.

Reply:
done